.github/workflows/ubuntu-24.04-packaging-aarch64.yml

```diff
@@ -90,3 +90,43 @@ jobs:
           path: |
             ${{ github.workspace }}/zipfile/*
           retention-days: 1
+
+  install-test:
+    needs: build
+    runs-on: ubuntu-24.04-arm
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: jrei/systemd-ubuntu:22.04
+          - os: jrei/systemd-ubuntu:24.04
+          - os: jrei/systemd-debian:12
```

Comment:
Is this a dependency we want to bring into our supply chain?

Reply:
We could mitigate using a TAG, but then someone needs to update it.
We are not using this dependency to create the debian packages. 
Changing to